### ISSUE TITLE
npx why-npm-i-so-long does not work

### ISSUE THREAD

**Issue by jeetiss** [antonk52/why-npm-i-so-long#1]
Seems like you forgot to publish package to npm 🙃
```
 % npx why-npm-i-so-long ./package.json 
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/why-npm-i-so-long - Not found
npm ERR! 404 
npm ERR! 404  'why-npm-i-so-long@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

**Comment by antonk52** [comment]
Hi, thank you for the issue. I indeed tried to publish it, but it was marked as spam. I’ve emailed npm support and awaiting for their response. If it won’t go through I’m going to rename the repository and the package and publish it under a different name. We will see =)

**Comment by antonk52** [comment]
I got the publish access for the name and have published `1.0.0` version. You should be able to install it or/and run it using npx now 🎉

**Comment by jeetiss** [comment]
Yes, it is working now
thanks for tool 🤟🏻
